go/ql/test/query-tests/Security/CWE-798/jwt.go

```diff
@@ -139,60 +138,56 @@ func gogfjwt() interface{} {
 }
 
 func irisjwt() interface{} {
-	mySecret := []byte("key12")
+	safeName := []byte("key12")
 	token := iris.NewTokenWithClaims(nil, nil)
-	tokenString, _ := token.SignedString(mySecret)
+	tokenString, _ := token.SignedString(safeName) // BAD
```

Comment:
safeName suggests this isn't in fact bad?

Reply:
I renamed `mySecret` to `safeName` because I didn't want the other part of the query (`sensitiveAssignment`) to catch it. But independently of the name, a hardcoded string flowing to `SignedString` is a problem, and that's what I want to test.

Reply:
I opted for using a more neutral name (`key`) which is ambiguous enough to not be picked up by the heuristic and less confusing at the same time: https://github.com/github/codeql/pull/15615